### PR TITLE
[ACS-4051] CSV issue 880196 Function cannot be performed by keyboard alone addr…

### PR DESCRIPTION
**Pull request by jatin2008**
…essed

**Please check if the PR fulfills these requirements**

> - [x ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Function cannot be performed by keyboard alone

**What is the new behaviour?**

Now Function can be performed by keyboard alone, added tabindex to copy button

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by AleksanderSklorz — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Review by rbahirsheth — APPROVED** [review]

File: e2e/core/pages/notification.page.ts [hunk 1]
```diff
@@ -46,18 +46,6 @@ export class NotificationDemoPage {
         return BrowserActions.getText(this.notificationConfig);
     }
 
-    async isNotificationSnackBarDisplayed(): Promise<boolean> {
```
Comment (AleksanderSklorz) [inline e2e/core/pages/notification.page.ts:49]: @jatin2008 I see you added some files in last commit which probably should not be here?

**Review by MichalKinas — CHANGES_REQUESTED** [review]
@jatin2008 this merge needs to be reverted, our policy says that we should update our branches using rebase not merge

File: e2e/core/pages/notification.page.ts (hunk 1 above)
Comment (MichalKinas) [reply e2e/core/pages/notification.page.ts:49]: Yeah, that's why we all should use rebase instead of merge to update branches

**Comment by jatin2008** [comment]
merging issue hence closing this PR

**Pull request closed without merging**